compiler/util/files.cpp

```diff
@@ -424,13 +425,48 @@ const char* createDebuggerFile(const char* debugger, int argc, char* argv[]) {
   return dbgfilename;
 }
 
-const std::string runUtilScript(const char* script) {
+std::string runPrintChplEnv(std::map<std::string, const char*> varMap) {
+  // Run printchplenv script, passing currently known CHPL_vars as well
+
+  std::string command = "";
+  std::string result = "";
+  char buffer[256];
+
+  // Pass known variables into printchplenv via command
+  for( std::map<std::string, const char*>::iterator ii=varMap.begin(); ii!=varMap.end(); ++ii)
+  {
+    command += ii->first + "=" + std::string(ii->second) + " ";
```

Comment:
I don't check it here, because there is never a case where I call `runPrintChplEnv` with a varMap that has `CHPL_THREADS` defined. I could add it in as a safety precaution though.